docs/src/wallet_to_wallet_with_tariscript.md

```diff
@@ -0,0 +1,205 @@
+
+\\( \hat{C}_c = (k_c + \beta_c) \cdot G + v_c \cdot H \\)
+
+where
+
+\\( \beta_c = H(\sigma_c || F_c || K_{Oc}) \\).
+
+Alice then sends the following values to Bob:
+
+| Symbol  | Description                                      |
+|---------|--------------------------------------------------                                   |
+| \\( R_a \\)       | Alice's Public Nonce                                                      |
+| \\( v_b \\)       | Amount being sent to Bob                                                  |
+| \\( f \\)         | fee                                                                       |
+| \\( m \\)         | Transaction metadata (currently just lockheight)                          |
+| \\( \sigma_b \\)  | Hash of \\( C_b \\)'s spending script,  \\( \sigma_b = H(\alpha_b) \\)    |
+| \\( K_{Ob} \\)    | Public UTXO script offset                                                 |
+| message           | A unicode string                                                          |
+
+### Bob Replies
+Once Bob receives these values he will choose the following:
+1. Spending key \\( k_b \\) of \\( C_b \\)
+2. A random nonce \\( r_b \\) whose public value is \\( R_b = r_b \cdot G \\)
+3. His script key \\( k_s \\) whose public value \\( K_s = k_s \cdot G \\) must be the final value on the stack after script \\( alpha_b \\) executes.
+4. Output features \\( F_b \\) for \\( C_b \\) if any.
+
+Bob produces the rangeproof \\( RP_b \\) for the modified commitment
+
+\\( \hat{C}_b = (k_b + \beta_b) \cdot G + v_b \cdot H \\) 
+
+where 
+
+\\( \beta_b = H(\sigma_b || F_b || K_{Ob}) \\).
+
+Bob then produces his partial signature of the transaction as per vanilla MW:
+
+\\( s_b = r_b + ek_b \\)
+
+where
+
+\\( e = H(R_a + R_b || f || m) \\)
+
+Bob then sends the following back to Alice:
+
+| Symbol  | Description                                      |
+|---------|--------------------------------------------------|
+| \\( C_b \\)       | Bob's commitment                      |
+| \\( K_b \\)       | Bob's public spending key             |
+| \\( F_b \\)       |  The Output Features of \\( C_b \\)   |
+| \\( s_b, R_b \\)  | Bob's partial signature               |
+
+### Alice Round 2
+Alice now has the information to construct the challenge \\( e \\)to calculate her partial signature:
+
+\\( s_a = r_a + ek_a \\)
+
+and can produce the final kernel signature:
+
+\\( (s = s_a + s_b,  R = R_a + R_b )\\)
+
+Alice then produces a script signature (\\( s_{sa}, R_{sa} \\)) for spending \\( C_a \\) by signing \\( H( R_{sa} || \alpha_a || \theta_a || h) \\) with her private script key \\( k_{sa} \\).
+
+Alice can now calculate the script offset, \\( \gamma \\) for this transaction which will include the script of the input \\( C_a \\) and the two outputs \\( C_b \\) and \\( C_c \\).
+
+\\( \gamma =  k_{sa} - k_{Ob}U_b - k_{Oc}U_c \\)
+
+### Final transaction
+
+Alice can now construct the final transaction:
```

Comment:
I think it's worth adding the `offset` and `script_offset` at the transaction level to this table